src/components/pages/Flagging/ResolveFlagMember.tsx

```diff
@@ -0,0 +1,271 @@
+  flagToResolveOperation: FlagMemberOperation;
+  flaggingMember: Member;
+  apiCallId: string;
+  resolveFlagApiCallStatus?: ApiCallStatus;
+}
+
+interface DispatchProps {
+  resolveFlagMember: (
+    memberId: MemberId,
+    flagOperationId: OperationId,
+    reason: string,
+    apiCallId: string
+  ) => void;
+}
+
+type Props = OwnProps & StateProps & DispatchProps;
+
+interface State {
+  reviewedInfo: boolean;
+  reason?: string;
+}
+
+class ResolveFlagMemberPageComponent extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      reviewedInfo: false
+    };
+  }
+
+  continue = () => {
+    this.setState({ reviewedInfo: true });
+  };
+
+  renderInfo = () => {
+    return (
+      <React.Fragment>
+        <View style={sharedStyles.section}>
+          <Text style={sharedStyles.infoText}>
+            Resolving this flag will indicate that you believe this issue has
+            been fixed.
+          </Text>
+        </View>
+        <View style={sharedStyles.section}>
+          {this.props.isOwnProfile ? (
+            <Text style={sharedStyles.error}>
+              You cannot resolve a flag on your own profile. Make sure you've
+              addressed the issue, and then ask another Raha member to resolve
+              the flag for you.
+            </Text>
+          ) : this.props.canFlag ? (
+            <Button
+              title="Continue"
+              onPress={this.continue}
+              style={sharedStyles.button}
+            />
+          ) : (
+            <Text style={sharedStyles.error}>
+              You must be verified by at least 5 other Raha members to resolve
+              this flag.
+            </Text>
+          )}
+        </View>
+      </React.Fragment>
+    );
+  };
+
+  flagMember = async () => {
+    this.props.resolveFlagMember(
+      this.props.memberOnWhichToResolveFlag.get("memberId"),
+      this.props.flagToResolveOperation.id,
+      this.state.reason as string,
+      this.props.apiCallId
+    );
+  };
+
+  renderResolveFlagForm = (flaggedMemberName: string) => {
+    const { resolveFlagApiCallStatus } = this.props;
+    const disableResolveFlagButton =
+      !!resolveFlagApiCallStatus &&
+      resolveFlagApiCallStatus.status !== ApiCallStatusType.FAILURE;
+    const resolvedFlag =
+      !!resolveFlagApiCallStatus &&
+      resolveFlagApiCallStatus.status === ApiCallStatusType.SUCCESS;
+    const resolveFlagButtonTitle = disableResolveFlagButton
+      ? resolvedFlag
+        ? "Resolved flag"
+        : "Resolving flag"
+      : "Resolve flag";
+
+    return (
+      <React.Fragment>
+        <View style={sharedStyles.section}>
+          <Text style={sharedStyles.infoHeader}>
+            How has this issue with{" "}
+            <Text style={sharedStyles.name}>{flaggedMemberName}</Text>
+            's account been fixed?
+          </Text>
+        </View>
+        <View style={sharedStyles.section}>
+          <TextInput
```

Comment:
nit: Doesn't have to be fixed in this PR, but it's slightly confusing to me that the TextInput box looks so similar to flag notice box